assets/src/stories-editor/components/font-family-picker/index.js

```diff
@@ -24,44 +30,60 @@ function FontFamilyPicker( {
 	onChange = () => {},
 	value = '',
 } ) {
-	const defaultOption = {
-		value: '',
-		label: __( 'None', 'amp' ),
+	const results = fonts;
+	const suggest = ( query, syncResults ) => {
+		const searchResults = query ? results.filter( function( result ) {
+			return result.name.toLowerCase().indexOf( query.toLowerCase() ) !== -1;
+		} ) :
+			[];
+		syncResults( searchResults );
 	};
 
-	const options = fonts.map( ( font ) => ( {
-		value: font.name,
-		label: font.name,
-	} ) );
+	const suggestionTemplate = ( result ) => {
+		maybeEnqueueFontStyle( result.name );
```

Comment:
This loads the whole font stylesheet, whereas for performance reasons it might be beneficial to only load the glyphs actually needed to display the font name.

See https://github.com/ampproject/amp-wp/issues/3027#issuecomment-526531210 for an example.

Then, when the font has been selected, the whole font stylesheet could be loaded.

Reply:
I did some profiling here, the font is only 24kb, which is the size of a small image. As this is only loaded once the font is being displayed, it also has the benefit the preload the font before when displayed in the content of the amp page. Even 10 fonts loaded is only 240kb, which is the size of a small image.

Reply:
Hmm that sounds promising. I'll check it out myself while reviewing.